code/android-core-library/src/main/java/com/adobe/marketing/mobile/launch/rulesengine/json/JSONCondition.kt

```diff
@@ -38,30 +39,34 @@ internal abstract class JSONCondition {
          * @return a subclass of [JSONCondition]
          */
         @JvmSynthetic
-        internal fun build(jsonCondition: JSONObject?): JSONCondition? {
+        internal fun build(jsonCondition: JSONObject?, extensionApi: ExtensionApi): JSONCondition? {
             if (jsonCondition !is JSONObject) return null
             return try {
                 when (val type = jsonCondition.getString(KEY_TYPE)) {
                     TYPE_VALUE_GROUP -> GroupCondition(
                         JSONDefinition.buildDefinitionFromJSON(
                             jsonCondition.getJSONObject(
                                 KEY_DEFINITION
-                            )
+                            ),
+                            extensionApi
                         )
                     )
                     TYPE_VALUE_MATCHER -> MatcherCondition(
                         JSONDefinition.buildDefinitionFromJSON(
                             jsonCondition.getJSONObject(
                                 KEY_DEFINITION
-                            )
+                            ),
+                            extensionApi
```

Comment:
Has `MatcherCondition` and `GroupCondition` been changed to accept `extensionApi`? Don't see it in this PR

Reply:
No, we don't need to add it to other conditions as they don't depend on `ExtensionApi`.